yarn.lock

```diff
@@ -4654,6 +4654,18 @@
     base64-sol "1.0.1"
     hardhat-watcher "^2.1.1"
 
+"@uniswap/v3-periphery@^1.2.1":
```

Comment:
hm can we make sure there's only one of these?

Reply:
The multicall lib is using a newer version of v3-periphery than the interface is. I can downgrade its version to match or we can upgrade the interface. @moodysalem what do you recommend?

Reply:
upgrade the interface, don't think there's any breaking changes (at least that affect the interface)

Reply:
I've made multicall + interface consistent on 1.3.0 but the @uniswap/v3-sdk uses an old periphery version (1.1.1). 
I've added a yarn resolution for now. Alternatively we can upgrade the v3-sdk and ship an update.